charts/posthog/values.yaml

```diff
@@ -746,6 +746,108 @@ recordingsIngestion:
   # -- Sentry endpoint to send errors to. Falls back to global sentryDSN
   sentryDSN:
 
+recordingsBlobIngestion:
+  # -- Whether to install the PostHog session recordings blob ingestion capability as
+  # an individual workload.
+  enabled: false
+
+  # -- Count of plugin-server pods to run. This setting is ignored if `recordingsBlobIngestion.hpa.enabled` is set to `true`.
+  replicacount: 1
+
+  hpa:
+    # -- Whether to create a HorizontalPodAutoscaler for the plugin stack.
+    enabled: false
+    # -- CPU threshold percent for the plugin-server stack HorizontalPodAutoscaler.
+    cputhreshold: 60
+    # -- Min pods for the plugin-server stack HorizontalPodAutoscaler.
+    minpods: 1
+    # -- Max pods for the plugin-server stack HorizontalPodAutoscaler.
+    maxpods: 10
+    # -- Set the HPA behavior. See
+    # https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+    # for configuration options
+    behavior:
+
+  keda:
+    # -- Whether to create a ScaleObject to handle scaling of the consumer group
+    # deployment using KEDA. `hpa.enabled` must be false to use this. We will
+    # use the same configuration to connect to Kafka as either the ones
+    # generated by the chart if using `kafka.enabled=true` or the config from
+    # `externalKafka` otherwise.
+    enabled: false
+
+    # -- Config added to spec at the top level of the `ScaledObject`.
+    config:
+      pollingInterval: 30
+
+    # -- KEDA trigger config for type kafka. This will be merged in with other
+    # config such as consumer group id and topic which will be set
+    # automatically.
+    #
+    # TODO: add support for all security protocol config.
+    kafkaTrigger:
+      enabled: true
+
+      metadata:
+        # -- At what lag value should KEDA scaling start.
+        lagThreshold: "1000"
```

Comment:
These defaults will probably be slightly different but we can play with that later.

Reply:
Yeah true